JetUtilities/interface/L2Creator.hh

```diff
@@ -99,7 +99,7 @@ private:
     TString                   output, outputDir, l2calofit, l2pffit;
     vector<string>            formats, algs;
     bool                      l2l3, delphes;
-    int                       maxFitIter;
+    int                       maxFitIter, statTh;
```

Comment:
It wasn't immediately clear to me from the name what this variable was. Can you call it "statThreshold"?

Reply:
done